Test/Install-Service.Tests.ps1

```diff
@@ -45,11 +45,29 @@ function Assert-ServiceInstalled
     
 function Assert-HasPermissionsOnServiceExecutable($Identity, $Path)
 {
-    $access = Get-Permission -Path $Path -Identity $Identity
+    $access = Get-CPermission -Path $Path -Identity $Identity
     $access | Should Not BeNullOrEmpty
     ([Security.AccessControl.FileSystemRights]::ReadAndExecute) | Should Be ($access.FileSystemRights -band [Security.AccessControl.FileSystemRights]::ReadAndExecute)
 }
 
+function Assert-HasNoPermissionsOnServiceExecutable($Identity, $Path)
+{
+    $access = Get-CPermission -Path $Path -Identity $Identity
+    $access | Should BeNullOrEmpty

```

Comment:
Use Pester 4+ `Should` syntax: all the `Be*` operators are now switches/parameters, so add the `-` to each, e.g. `Should -BeNullOrEmpty`.

Reply:
Done. Updated all for Be* operators to the switches/parameters syntax for all the Install-Service.Tests.ps1.